### DISCUSSION TITLE
Question for port type list

### DISCUSSION THREAD

**Discussion by Eraden** [Eraden/amdgpud#47] [Q&A]
Hi everyone

I'm currently building GPU Card ports list with status and maybe (depends how hard and time consuming it would be) list of resolution.

But I need to ask for output of fallowing command and information what kind of port do you have:

```
ls /sys/class/drm | grep -E 'card[0-9]*-[a-zA-Z]*'
``` 

It will just print list of card number, port type and port number

Example:

```
card0-DP-1
card0-DP-2
card0-HDMI-A-1
card0-HDMI-A-2
```

**Comment by sa7dse** [comment]
card0-DP-1
card0-DP-2
card0-DP-3
card0-HDMI-A-1

**Comment by edwinvanolst** [comment]
card0-DP-1
card0-DVI-D-1
card0-DVI-D-2
card0-HDMI-A-1

**Comment by IkeBroflovskiForPresident** [comment]
card0-DP-1
card0-DP-2
card0-DP-3
card0-DVI-D-1
card0-HDMI-A-1

and

card0-DP-1
card0-DVI-D-1
card0-HDMI-A-1
